internal/api/request.go

```diff
@@ -69,53 +45,80 @@ func (client *Client) doRequest(ctx context.Context, token *string, request *htt
 		ua := client.buildUserAgent(ctx)
 		request.Header.Set("User-Agent", ua)
 
-		rid, cc := client.BuildCorrelationHeaders(ctx)
+		rid, cc := client.buildCorrelationHeaders(ctx)
 		request.Header.Set("Request-Id", rid)
```

Comment:
Do we need rid and cc headers as they are not logged in the backend?

Reply:
Updated it to use what PPAC uses.  Hopefully those come through